server/routes/api.js

```diff
@@ -33,6 +35,7 @@ router.use('/reader', authMiddleware('admin reader', 'api'), readerHandler);
 router.use('/push', pushHandler);
 router.use('/scan', scanHandler);
 router.use('/sponsor', sponsorHandler);
+router.use('/location', locationHandler), router.use('/event', eventHandler);
```

Comment:
Let's separate these

Reply:
@antoniomika i'm not sure why but prettify keeps turning this into one line -- not sure what to do about that

Reply:
Even where there is a `;` in between?